### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by 15mikesmith** [ajolivet/pwp-capstones#2]
In the future I would definitely including more comments in your code. Normally, as best practice, developers leave a comment for each method, to describe the purpose, inputs/outputs, etc.

**Comment by ajolivet** [comment]
> 
> 
> In the future I would definitely including more comments in your code. Normally, as best practice, developers leave a comment for each method, to describe the purpose, inputs/outputs, etc.

Thanks for the feedback @15mikesmith , I added comments, not to every method (as I think some of them are pretty self-explanatory) but to the ones which at least seemed non-trivial, as well as for each class. I'm new to this github interface, I followed the tutorial a long time ago, but it's the first time I'll try to update a project for real, so if I understood correctly, I should open a new branch, commit the changes to this branch and then open a pull request, then merge the new branch? Is that it? How does that relate to this issue, should I just close it once I fixed it with the merge?
